src/generic_core/core.ts

```diff
@@ -25,7 +25,7 @@ var loggingProvider = freedom['loggingprovider']();
 loggingProvider.setConsoleFilter(['*:I']);
 loggingProvider.setBufferedLogFilter(['*:D']);
 
-declare var UPROXY_VERSION;
+declare var UPROXY_VERSION :Object;
```

Comment:
`{[:string]:string}` is the best type it conforms to at the moment, I think just Object expresses what it needs to conform to for our use better however.